src/challenges.js

```diff
@@ -1,49 +1,125 @@
 // Desafio 1
-function compareTrue() {
-  // seu código aqui
+function compareTrue(value1, value2) {
+  if (value1 === true && value2 === true) {
+    return true;
+  } else {
+    return false;
+  }
 }
 
 // Desafio 2
-function calcArea() {
-  // seu código aqui
+function calcArea(base, height) {
+  return (base * height) / 2;
 }
 
 // Desafio 3
-function splitSentence() {
-  // seu código aqui
+function splitSentence(frase) {
+  let separada = frase.split(' ');
+  return separada;
 }
 
 // Desafio 4
-function concatName() {
-  // seu código aqui
+function concatName(allNames) {
+  let lastName = allNames[allNames.length - 1];
+  let firstName = allNames[0];
+  return lastName + ', ' + firstName;
 }
 
 // Desafio 5
-function footballPoints() {
-  // seu código aqui
+function footballPoints(wins, ties) {
+  let totalPoints = (wins * 3) + ties;
+  return totalPoints;
```

Comment:
O que acha de colocar esse código em uma linha só? Não ficaria muito mais difícil de entender, ficaria?

```suggestion
  return (wins * 3) + ties;
```